src/vi/ui/viui.py

```diff
@@ -42,7 +43,7 @@
 from vi.ui.systemtray import TrayContextMenu
 from PyQt5.QtCore import QSettings
 
-OLD_STYLE_WEBKIT = False
+OLD_STYLE_WEBKIT = "OLD_STYLE_WEBKIT" in os.environ
```

Comment:
This allows you to start it with "OLD_STYLE_WEBKIT=1 python3 vintel.py" on Linux and have it work, or set that environment variable on any other OS to use WebView instead of WebEngine.